modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -274,7 +274,7 @@ public static class Client {
         /** Client handshake header error. */
         public static final int HANDSHAKE_HEADER_ERR = CLIENT_ERR_GROUP.registerErrorCode((short) 8);
 
-        /** Server -> client request failed. */
+        /** Server -&gt; client request failed. */
```

Comment:
I assume we can avoid special characters at all.
```suggestion
        /** Server to client request failed. */
```